marketing.md

```diff
@@ -0,0 +1,32 @@
+# Marketing (and sales)
+
+## Sales and marketing
+
+This page concerns sales in the sense of bringing in new projects,
+but is technically more concerned with marketing to attract clients organically.
+When members find themselves needing work, it is natural for them to shift into _sales mode_.
+However, here we talk about ongoing activity to avoid such situations.
```

Comment:
I think this is a awesome clarification of the different aspects of what we are doing. 

My suggestion would be to:
1. change the name of this (and the drive to) `sales_and_marketing.md`
2. Either just have the title `# Sales and marketing` or something like the below. I might be inclinded to add the definitions since it helps to clarify and remind us.
```
# Sales and marketing

**Sales**: closing deals and generating revenue through direct interaction with customers
**Marketing**: a broader approach, creating awareness, building relationships, and driving long-term sales growth
```